src/pages/home/report/ReportActionItem.js

```diff
@@ -346,7 +346,7 @@ function ReportActionItem(props) {
             const iouReportID = originalMessage.IOUReportID ? originalMessage.IOUReportID.toString() : '0';
             children = (
                 <MoneyRequestAction
-                    chatReportID={props.report.reportID}
+                    chatReportID={originalMessage.IOUReportID ? props.report.chatReportID : props.report.reportID}
```

Comment:
Can you just add a comment explaining this check so the next person knows why we needed to do this?

Reply:
Please ensure that this is in proper English. You can use freely available tools like ChatGPT 3.

Reply:
Try this

> If originalMessage.iouReportID is set, this is a 1:1 money request in a DM chat whose reportID is props.report.chatReportID